example/settings.py

```diff
@@ -19,7 +19,7 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
-SECRET_KEY = "mysecret"
+SECRET_KEY = "231cc7b8-63ce-4a87-98ea-4c663ee9557d:d7bcde56-e812-4ef0-aa07-3b22f3066d2f"
```

Comment:
This is just for an example app so does not matter but fixed a django check warning I was seeing.